### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by aneeshkp**
Signed-off-by: Aneesh Puttur <aneeshputtur@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,71 @@
-# hw-event-proxy-operator
+# Hardware Event proxy Operator
```
Comment (jzding) [inline README.md:1]: proxy -> Proxy

File: README.md [hunk 2]
```diff
@@ -1 +1,71 @@
...
+
+
+## HardwareEvent
+To start using Hardware event proxy, install the operator and then create the following HardwareEvent:
+You can only create one instance of HardwareEvent in this release.
+```
+apiVersion: "event.redhat-cne.org/v1alpha1"
+kind: "HardwareEvent"
+metadata:
+name: "hardwareevent"
```
Comment (jzding) [inline README.md:21]: need indent for name

File: README.md [hunk 3]
```diff
@@ -1 +1,71 @@
-# hw-event-proxy-operator
+# Hardware Event proxy Operator
+## Table of Contents
+
+- [HW-Event-Proxy Operator](#hw-event-proxy operator)
```
Comment (jzding) [inline README.md:4]: Need a dash here for the link to work: #hw-event-proxy-operator

File: README.md [hunk 4]
```diff
@@ -1 +1,71 @@
...
+metadata:
+name: "hardwareevent"
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/hw-event: ""
+  transportHost: "amqp://amq-router-service-name.amq-namespace.svc.cluster.local"
+```
+
+## Quick Start
+
```
Comment (jzding) [inline README.md:62]: add prerequisite/instructions for installing operator-sdk?

File: README.md (hunk 1 above)
Comment (aneeshkp) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (aneeshkp) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (aneeshkp) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (aneeshkp) [reply README.md:62]: it is standard for all operators . Don't have to mention it.

**Review by jzding — APPROVED** [review]

**Pull request merged**